src/lib/components/forms/form/form-components/text-input/text-input.component.spec.ts

```diff
@@ -21,7 +24,17 @@ describe('TextInputComponent', () => {
     beforeEach(async () => {
         TestBed.configureTestingModule({
             imports: [MatFormFieldModule, MatInputModule, NoopAnimationsModule, FormsModule],
-            declarations: [TextInputComponent, MockTooltipDirective]
+            declarations: [TextInputComponent, MockTooltipDirective],
```

Comment:
Die Directive für den Iban-Validator fehlt noch
```suggestion
            declarations: [TextInputComponent, MockTooltipDirective, IbanValidatorDirective],
```

Reply:
bin noch dran, wenn ich sie hinzufüge bekomme ich einen timeout error:
 Error: Timeout - Async function did not complete within 5000ms

Reply:
Hmm. Aber die Directive muss da auf jeden Fall hinzugefügt werden.

Reply:
habe sie erstmal so blind hinzugefügt, um zu schauen, ob der Check so geht!

Reply:
Dir fehlt noch das `L10nTranslationModule` in den Imports
<img width="749" alt="Bildschirmfoto 2021-04-22 um 09 30 04" src="https://user-images.githubusercontent.com/26869118/115673877-622f2c00-a34d-11eb-9c13-0876ea9ff9b2.png">

Reply:
ah ja, Übeltäter identifiziert! :)